gcp/deployment-packages/autoscale-byol/c2d_deployment_configuration.json

```diff
@@ -1,6 +1,6 @@
 {
   "defaultDeploymentType": "MULTI_VM",
-  "imageName": "check-point-r8110-gw-byol-mig-335-991001174-v20221113",
+  "imageName": "check-point-r8120-gw-byol-mig-631-991001475-v20231221",
```

Comment:
r8120-gw-byol-mig-631-991001475 
Please verify the build number (1475) in all places